elide-core/src/main/java/com/yahoo/elide/core/pagination/Pagination.java

```diff
@@ -80,9 +89,18 @@ public static Pagination parseQueryParams(final MultivaluedMap<String, String> q
                         } catch (ClassCastException e) {
                             throw new InvalidValueException("page values must be integers");
                         }
+                    } else if (queryParamKey.equals(PAGE_TOTAL_KEY)) {
+                        final String value = paramEntry.getValue().get(0);
+                        if (value.equals("true")) {
```

Comment:
It would seem a bit odd if "page[totals]" value of "false" or "no" was passed and the page totals were still generated.

Also, I am thinking that the client will specify "true" only for the first page request, then with subsequent page requests the totals will normally not need to be recalculated. There is some convenience with being able to just change the value to "false" to stop the totals recalculation.